libraries/AP_RangeFinder/AP_RangeFinder_Ainstein_LR_D1.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "AP_RangeFinder_config.h"
+
+/*
+ questions:
+ - bug on page 22 on malfunction codes
+ - fixed length seems strange at 28 bytes
+ - definition of snr field is missing in documentation
+ - roll/pitch limits are in conflict, 3.2 vs 
+*/
+
```

Comment:
There should be a` #if AP_RANGEFINDER_AINSTEIN_LR_D1_ENABLED here`

Reply:
added